thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/template.hbs

```diff
@@ -1,24 +1,27 @@
-<ul class="rootcause-metric__display-view">
-  <li class="rootcause-metric__display-view--{{if (eq selectedView "card") "active"}}" {{action "selectView" 'card'}}>
-    Card
-  </li>
-  <li>|</li>
-  <li class="rootcause-metric__display-view--{{if (eq selectedView "table") "active"}}" {{action "selectView" 'table'}}>
-    Table
-  </li>
-</ul>
-
-{{#if (eq selectedView "card")}}
-  <div class="row">
-    <div class="col-xs-12">
+<div class="row">
+  {{#if (eq selectedView "card")}}
+    <div class="col-xs-8">
       sort
-      | <button {{action "toggleSort" "metric"}}>metric</button>
-      | <button {{action "toggleSort" "dataset"}}>dataset</button>
-      | <button {{action "toggleSort" "change"}}>change</button>
-      | <button {{action "toggleSort" "score"}}>score</button>
+      | <a {{action "toggleSort" "metric"}}>metric</a>
+      | <a {{action "toggleSort" "dataset"}}>dataset</a>
+      | <a {{action "toggleSort" "change"}}>change</a>
+      | <a {{action "toggleSort" "score"}}>score</a>
     </div>
+  {{/if}}
+  <div class="col-xs-4 pull-right">
+    <ul class="rootcause-metric__display-view">
+      <li class="rootcause-metric__display-view--{{if (eq selectedView "card") "active"}}" {{action "selectView" 'card'}}>
```

Comment:
fyi, the correct way to use modifier with BEM is by using the default class + modifier class. For instance if we had a button we would see: 
```
<button class="te-button te-button--red">
```
The "correct" way would be to name your classes so that the class names are as follows: 
```
<li class="rootcause-metric__display-view rootcause-metric__display-view--active>
```

This can be fixed in a subsequent pr

Reply:
@ttbach

Reply:
Yves's right. My initial CSS wasn't BEM compliant. I would suggest doing the following:
```
<ul class="rootcause-metric__view-options">
 <li class="rootcause-metric__display-view rootcause-metric__display-view--{{if (eq selectedView "card") "active"}}" {{action "selectView" 'card'}}>
    Card
 </li>
 <li class="rootcause-metric__display-view rootcause-metric__display-view--{{if (eq selectedView "table") "active"}}" {{action "selectView" 'table'}}>
    Table
 </li>
</ul>
```